src/NetworkEquipment.php

```diff
@@ -173,10 +174,32 @@ public function prepareInputForAdd($input)
         unset($input['id']);
         unset($input['withtemplate']);
 
+        return $this->manageInput($input);
+    }
+
+
+    public function prepareInputForUpdate($input)
+    {
+        return $this->manageInput($input);
+    }
+
+
+    public function manageInput($input)
+    {
+        //without right, user can't set remote_addr
+        if (isset($input['remote_addr']) && !Session::haveRight(Conf::$rightname, Conf::IMPORTFROMFILE)) {
+            unset($input['remote_addr']);
+        }
         return $input;
     }
 
 
+    public static function getRemoteAddrLabel()
+    {
+        return __('SNMP port remote address');
```

Comment:
Should be "SNMP remote address" no? I mean , a port has no address :)

Reply:
yeap =)